<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunningServerUi</class>
 <widget class="QWidget" name="RunningServerUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>204</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QPushButton" name="startServer">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changePort">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Port</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="infoBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="portNumber">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>161</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>5000</number>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
